.serverless/cloudformation-template-create-stack.json

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
This file is auto generated and shouldn't be committed.
The entire .serverless folder shouldn't be committed.

Reply:
removed

Reply:
I mean, put it in gitignore, or does it stay in the repo then?